MoreLinq.Test/TestExtensions.cs

```diff
@@ -73,5 +73,9 @@ internal static IEnumerable<string> GenerateSplits(this string str, params char[
                 yield return split;
         }
 
+        internal static void Add<TKey, TValue>(this List<KeyValuePair<TKey, TValue>> list, TKey key, TValue value)
```

Comment:
If we are going to add this, might as well extend `IList<KeyValuePair<TKey, TValue>>` (_interface_) as opposed to `List<KeyValuePair<TKey, TValue>>` (_implementation_).